Source/WTF/wtf/FastMalloc.h

```diff
@@ -656,15 +656,26 @@ void operator delete(T* object, std::destroying_delete_t, size_t size) { \
 } \
 using WTFDidOverrideDeleteForCheckedPtr = int;
 
+#define WTF_CHECKEDPTR_NO_NONNULL
```

Comment:
How about `#define WTF_NULLABLE _Nullable` to avoid double-negative and make it reusable for non-CheckedPtr macros?

Reply:
Or at that point could we just pass `_Nullable` to our callee macro?